### PR TITLE
11.0 mig l10n it corrispettivi

### PR DESCRIPTION
**Pull request by SimoRubi**
Migration for l10n_it_corrispettivi

### CODE REVIEW

**Comment by SimoRubi** [comment]
@dcorio which email is associated to your Github account? I need to set that in https://github.com/OCA/l10n-italy/pull/585/commits/f582dbc45ace6f0145f53d4626ed9287171bf0dc

**Comment by SimoRubi** [comment]
Created https://github.com/OCA/l10n-italy/pull/586 to fix runbot

**Comment by SimoRubi** [comment]
Rebased after https://github.com/OCA/l10n-italy/pull/586

**Comment by SimoRubi** [comment]
Backported fix for `default_journal` to v10 in https://github.com/OCA/l10n-italy/pull/592

**Comment by SimoRubi** [comment]
@OCA/local-italy-maintainers could you please check? Thanks

File: l10n_it_corrispettivi/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    'name': 'Italian Localization - Corrispettivi',
+    'version': '11.0.1.0.0',
```
Comment (tafaRU) [inline l10n_it_corrispettivi/__manifest__.py]: please add https://docs.google.com/document/d/1MmGxv0yTw0pvB-PYAcUhbCcku9ZG-Y2DSGXwZDNYyIw according to https://docs.google.com/document/d/1MmGxv0yTw0pvB-PYAcUhbCcku9ZG-Y2DSGXwZDNYyIw

Comment (SimoRubi) [reply l10n_it_corrispettivi/__manifest__.py]: Done in https://github.com/OCA/l10n-italy/pull/585/commits/befb42f6223940b74cdd0e8490e57dde06633e0c

Comment (tafaRU) [reply l10n_it_corrispettivi/__manifest__.py]: :+1:  I also checked https://github.com/SimoRubi/l10n-italy/blob/11.0-mig-l10n_it_corrispettivi/l10n_it_corrispettivi/README.rst and it doesn't need to be updated, probably 'cause Beta is the default for `development_status`.

**Comment by SimoRubi** [comment]
Fixed up commits, @OCA/local-italy-maintainers could you please check? Thanks

**Comment by SimoRubi** [comment]
Fixed commit author for https://github.com/OCA/l10n-italy/commit/816a54c5aaff4b2e8b5e77e76ec8652caaec80f5 (now https://github.com/OCA/l10n-italy/pull/585/commits/cd1b2a2807d115729d855efbedb2d8a807c6f086)

**Comment by eLBati** [comment]
Resta solo da spostare il menu corrispettivi sotto 'documents'

**Comment by SimoRubi** [comment]
@eLBati done in https://github.com/OCA/l10n-italy/pull/585/commits/187de9a9999510b21f416482459f6c490789fb30

**Pull request merged**